packages/react/src/useFocusOut.js

```diff
@@ -53,7 +53,6 @@ function useFocusOut(ref, callback, deps) {
       node.removeEventListener('focusout', onFocusOut);
       doc.removeEventListener('pointerdown', onDocumentClick, opts);
     };
-    // eslint-disable-next-line react-hooks/exhaustive-deps -- Pass through provided deps.
```

Comment:
We should update `additionalHooks` in the ESLint config so we can keep this

Reply:
I didn't remove this line, not sure why eslint removes it automatically

<img width="791" alt="image" src="https://user-images.githubusercontent.com/6297436/163414888-1ff0e122-327e-4f37-bc25-8b739ff9c97a.png">

Reply:
I think this is why

<img width="945" alt="image" src="https://user-images.githubusercontent.com/6297436/163415102-9d441a8c-6a42-465d-859c-d4c96041bc05.png">

Reply:
After updating the `additionalHooks` config this should not happen anymore and we can add it back

Reply:
Oh I see, updated in https://github.com/GoogleForCreators/web-stories-wp/pull/11270/commits/843e32a8465dc8b8643c68345c9d9f44aa524bc7